Source/LuaBridge/detail/LuaRef.h

```diff
@@ -301,6 +301,18 @@ class LuaRefBase
     return Stack <T>::get (m_L, -1);
   }
 
+  //----------------------------------------------------------------------------
+  /**
+      Type check
+  */
+  template <class T>
+  bool isInstance() const
```

Comment:
Could you please fix the coding style?
There should always be spaces around any brackets.

Reply:
Certainly. I've comitted it to my fork. I am new to GitHub, how can I update the pull request?

Reply:
Oh I see its automatic.